### PR TITLE
chore: moved viewport meta from document to PageWrapper

### LINKED ISSUE #9405: [CAL-1917] chore: meta viewport in _app not _document
```
import Head from "next/head";

import { trpc } from "@calcom/trpc/react";

import type { AppProps } from "@lib/app-providers";

import "../styles/globals.css";

function MyApp(props: AppProps) {
  const { Component, pageProps } = props;
  if (Component.PageWrapper !== undefined) return Component.PageWrapper(props);
  return (
    <>
      <Head>
        <meta
          name="viewport"
          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
        />
      </Head>
      <Component {...pageProps} />
    </>
  );
}

export default trpc.withTRPC(MyApp);

```

<sub>[CAL-1917](https://linear.app/calcom/issue/CAL-1917/chore-meta-viewport-in-app-not-document)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
fixes https://nextjs.org/docs/messages/no-document-viewport-meta

### CODE REVIEW

File: apps/web/pages/_app.tsx [hunk 1]
```diff
@@ -7,7 +9,17 @@ import "../styles/globals.css";
 function MyApp(props: AppProps) {
   const { Component, pageProps } = props;
   if (Component.PageWrapper !== undefined) return Component.PageWrapper(props);
```
Comment (keithwillcode) [inline apps/web/pages/_app.tsx:11]: Do we not need to also do this in the case this line? i.e. When a page uses PageWrapper?

Comment (PeerRich) [reply apps/web/pages/_app.tsx:11]: not really sure to be honest

Comment (emrysal) [reply apps/web/pages/_app.tsx:11]: I think we do 👍

File: apps/web/pages/_app.tsx [hunk 2]
```diff
@@ -7,7 +9,17 @@ import "../styles/globals.css";
...
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+        />
+      </Head>
```
Comment (roae) [inline apps/web/pages/_app.tsx:19]: @PeerRich I think this should be in the PageWrapper Component in order to get the meta tag in all pages.

Only the calendar-cache page reach this code and this meta tag is useless there.

Comment (roae) [reply apps/web/pages/_app.tsx]: I have already did this change, I think the PR title must change

imagen

**Review by keithwillcode — APPROVED** [review]
Nice. Looks good.

**Pull request merged**